### DISCUSSION TITLE
Can the views be added to the following endpoints as well?

### DISCUSSION THREAD

**Discussion by priyam1103** [etsy/open-api#725] [Q&A] [answered]
- https://openapi.etsy.com/v3/application/listings/active
- https://openapi.etsy.com/v3/application/shops/{shop_id}/listings/active

**Comment by etsyachristensen** [comment, marked as answer]
The views should already be returning on both of those endpoints. If they are coming out a `0` it could be a timeout issue in our system. You'd have to fetch the listings individually to verify the view count. Alternatively, as a test, maybe try reducing the `limit` value on your call to those endpoints to 20 or so and see if it then returns the views value.

**Comment by priyam1103** [comment]
@etsyachristensen  Oh, I see, we are getting it. Sorry for the confusion actually this was not mentioned in the documentation on these endpoints so I assumed it's not there. All good now 👍
